recruiter/views.py

```diff
@@ -71,5 +71,22 @@ def recruiterRegister(request):
     return render(request, 'recruiter/registerRecruiter.html', context)
 
 
+def companyRegister(request):
+    if request.method == 'POST':
+        form = CompanyRegistrationForm(request.POST)
+        if form.is_valid():
+            form.save()
+            return redirect('/company_created_successfully')
+    else:
+        form = CompanyRegistrationForm()
+
+    context = {'form': form}
+    return render(request, 'recruiter/registerCompany.html', context)
+
+
+def company_created_successfully(request):
+    return render(request, 'recruiter/company_created_successfully.html')
+
+
 def recruiter_created_successfully(request):
-    return render(request, 'recruiter/recruiter_created_successfully.html')
+    return render(request, 'recruiter/company_created_successfully.html')
```

Comment:
I don't understand why is the `recruiter_created_successfully()` view is using the `company_created_successfully` template...

Reply:
will be fixed, a mistake.

Reply:
Done.